civitai-downloader-v2/src/core/stream/intermediate_file_manager.py

```
@@ -0,0 +1,284 @@
+#!/usr/bin/env python3
+"""
+Intermediate File Manager - 中間ファイル管理とストリーム処理
+API結果の中間保存とエラーリカバリ機能を提供
+"""
+
+import json
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)